apps/server/lib/lexical/server/project/progress/util.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Lexical.Server.Project.Progress.Util do
```

Comment:
Note: A module named util almost is certainly not what you want ;)

Reply:
So, Any better names? Naming is so hard for me.😅